### DISCUSSION TITLE
Testing out various model performances

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [coqui-ai/TTS#960] [Qs for Live Q&A] [answered]
Any other ways to evaluate this not using the alignment curves and the loss curves.  

Is there a better way to do it than the MOS . Tried out the TestAttention notebook but it simply brings an error of invalid notebook

**Comment by erogol** [comment, marked as answer]
(answered in the call)

**Reply by Wayne-wonderai** [reply]
Hi, I also see invalid notebook. Was there a solution for this?

**Reply by erogol** [reply]
which one?

**Reply by Wayne-wonderai** [reply]
I mean ```TestAttention```. But I already fixed it, thanks.

**Reply by KakoozaJerry** [reply]
@Wayne-wonderai , How did you fix it?

**Reply by erogol** [reply]
how about if you send a ✨PR✨
